src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -317,15 +408,22 @@ private Vector limitDirectionSlewRate(Vector spdVec) {
     return (new Vector(m_currentTranslationMag, 0)).rot(m_currentTranslationDirRadians);
   }
 
+  /**
+   * checks whether pathplanner paths should be flipped based on the current alliance
+   *
+   * @return whether pathplanner paths should be flipped
+   */
+  private boolean allianceCheck() {
+    var alliance = DriverStation.getAlliance();
+    if (alliance.isPresent()) {
+      return alliance.get() == DriverStation.Alliance.Red;
+    }
+    return (false);
```

Comment:
Why the parentheses here? Seems unnecessary?

Reply:
yeah, I think that code is copied from somewhere else, I'll fix it